docs/Bootloaders.md

```diff
@@ -452,78 +490,45 @@ The bootloader can be activated by pressing the reset button of the board twice.
 As soon as the bootloader is activated, the board appears as a USB flash drive
 onto which the klipper.bin file can be copied.
 
-### STM32F103/STM32F0x2 with CanBoot bootloader
-
-The [CanBoot](https://github.com/Arksine/CanBoot) bootloader provides an option
-for uploading Klipper firmware over the CANBUS.  The bootloader itself is
-derived from Klipper's source code.  Currently CanBoot supports the STM32F103,
-STM32F042, and STM32F072 models.
-
-It is recommended to use a ST-Link Programmer to flash CanBoot, however it
-should be possible to flash using `stm32flash` on STM32F103 devices, and
-`dfu-util` on STM32F042/STM32F072 devices.  See the previous sections in this
-document for instructions on these flashing methods, substituting `canboot.bin`
-for the file name where appropriate.  The CanBoot repository linked above provides
-instructions for building the bootloader.
-
-The first time CanBoot has been flashed it should detect that no application
-is present and enter the bootloader.  If this doesn't occur it is possible to
-enter the bootloader by pressing the reset button twice in succession.
-
-The `flash_can.py` utility supplied in the `lib/canboot` folder may be used to
-upload Klipper firmware.  The device UUID is necessary to flash.  If you do not
-have a UUID it is possible to query nodes currently running the bootloader:
-```
-python3 flash_can.py -q
-```
-This will return UUIDs for all connected nodes not currently assigned a UUID.
-This should include all nodes currently in the bootloader.
+## STM32F4 micro-controllers with DFU bootloader
```

Comment:
This section discusses hidflash - so changing this to say "DFU bootloader" is incorrect.

Reply:
I removed the bootloader in the header